experiments/opencog/opencog-neuralnet/cython/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+INCLUDE_DIRECTORIES(${PYTHON_INCLUDE_DIRS})
+
+# TODO: replace by cogutil include
```

Comment:
is this file a copy from some cogutil file?

Reply:
File itself is not a copy, this section is a modified copy and at the moment I am not sure it can be just simply replaced by something from cogutil. Cogutil macros doesn't allow additional include directories which is required here. I believe adding new macro to the cogutil and reusing it there is some separate task.